src/util/positioning.ts

```diff
@@ -217,7 +217,7 @@ export class Positioning {
       availablePlacementArr: Array<string>) {
     let html = document.documentElement;
     // check for left-bottom
-    if ((window.innerHeight || html.clientHeight) - hostElemClientRect.left >= targetElemClientRect.width) {
+    if ((window.innerWidth || html.clientWidth) - hostElemClientRect.left >= targetElemClientRect.width) {
```

Comment:
Hmm, I'm wondering why existing unit tests didn't catch it? Could you please try to add a test that fails when this fix is not in place?